plugins/services/src/js/components/modals/ServiceDestroyModal.js

```diff
@@ -103,7 +103,8 @@ class ServiceDestroyModal extends React.Component {
 
   getIsRightButtonDisabled() {
     return (
-      this.props.service.getId() !== this.state.serviceNameConfirmationValue
+      this.formatServiceName(this.props.service.getId()) !==
+      this.formatServiceName(this.state.serviceNameConfirmationValue)
```

Comment:
It looks like we can reuse [this `#getName` method](https://github.com/dcos/dcos-ui/blob/master/plugins/services/src/js/structs/Service.js#L18-L20) on the `Service` struct, what do you think?